pkg/api/register.go

```diff
@@ -70,6 +69,7 @@ func init() {
 	)
 
 	// Register Unversioned types
+	Scheme.AddKnownTypes("", &unversioned.ListOptions{})
```

Comment:
cc @deads2k, will this work after we register internal objects to their respective groups (e.g., internal object in Extensions group is registered using `Scheme.AddKnownTypes("Extensions/", ...)`)?